pkg/testdata/put_create_resource_with_path_param.yml

```diff
@@ -22,6 +22,57 @@ components:
           $ref: "#/components/schemas/a_string_prop"
 
 paths:
+  /v2/differentResource:
+    post:
+      operationId: createDifferentResource
+      requestBody:
+        content:
+          application/json:
+            schema:
+              $ref: "#/components/schemas/request_object_type"
+      responses:
+        "200":
+          description: The response will be a JSON object with a key called `action`.
+          content:
+            application/json:
+              schema:
+                $ref: "#/components/schemas/response_object_type"
+
+  /v2/differentResource/{some_id}:
```

Comment:
Note how this endpoint path has a PUT but also its parent endpoint path, i.e. `v2/differentResource` does have a POST endpoint, so the PUT endpoint under this endpoint must not be considered as a method of creating the resource. It's only for updating the resource.